fe/fe-core/src/test/java/com/starrocks/replication/ReplicationJobTest.java

```diff
@@ -96,7 +96,7 @@ public void setUp() throws Exception {
         partition.updateVersionForRestore(10);
         srcPartition.updateVersionForRestore(partition.getCommittedVersion() + 100);
 
-        job = new ReplicationJob("test_token", db.getId(), table, srcTable, GlobalStateMgr.getCurrentSystemInfo());
+        job = new ReplicationJob("test_token", "id", db.getId(), table, srcTable, GlobalStateMgr.getCurrentSystemInfo());
```

Comment:
```suggestion
        job = new ReplicationJob(null, "test_token", db.getId(), table, srcTable, GlobalStateMgr.getCurrentSystemInfo());
```
otherwise the fe ut may failed due to concurrent running

Reply:
all done, PTAL